socketio/src/client/client.rs

```diff
@@ -143,9 +143,10 @@ impl Client {
         let disconnect_packet =
             Packet::new(PacketId::Disconnect, self.nsp.clone(), None, None, 0, None);
 
-        self.socket.send(disconnect_packet)?;
+        let _ = self.socket.send(disconnect_packet);
```

Comment:
This would be a good location for logging, not sure if there's a plan for logging yet in this crate @1c3t3a

Reply:
There is no plan yet, but I am happy for ideas around it. For know this would exceed the scope of this PR, but maybe add a comment to remember at a later point?

Reply:
IMO https://docs.rs/tracing/latest/tracing/ at (mostly) debug level (NOT INFO). Like how https://github.com/hyperium/hyper/blob/5ec094caa5c999e6f919a2bc82f5f3b7d40c2d8a/src/proto/h2/client.rs does it.